### PR TITLE
The T210 training course is updated

### PR DESCRIPTION
**Pull request by PetrKorchagin**

### CODE REVIEW

File: Customization/T210/PhoneRepairShop_Code/PhoneRepairShop_Code/PhoneRepairShop_code.csproj [hunk 1]
```diff
@@ -49,7 +49,8 @@
     <Reference Include="PX.Objects">
       <HintPath>..\..\..\..\Bin\PX.Objects.dll</HintPath>
     </Reference>
-    <Reference Include="PX.Web.Customization">
+    <Reference Include="PX.Web.Customization, Version=1.0.0.0, Culture=neutral, PublicKeyToken=3b136cac2f602b8e, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
```
Comment (k-s-popova) [inline Customization/T210/PhoneRepairShop_Code/PhoneRepairShop_Code/PhoneRepairShop_code.csproj]: Do we need this change?

File: Customization/T210/PhoneRepairShop_Code/PhoneRepairShop_Code/RSSVRepairPriceMaint.cs [hunk 2]
```diff
@@ -4,6 +4,7 @@
 using PX.Data.BQL.Fluent;
 using PX.Objects.IN;
 using PX.Objects.CT;
+using static PhoneRepairShop.RSSVWarranty;
```
Comment (k-s-popova) [inline Customization/T210/PhoneRepairShop_Code/PhoneRepairShop_Code/RSSVRepairPriceMaint.cs]: Do we need this line?

File: Customization/T210/PhoneRepairShop_Code/PhoneRepairShop_Code/PhoneRepairShop_code.csproj (hunk 1 above)
Comment (PetrKorchagin) [reply Customization/T210/PhoneRepairShop_Code/PhoneRepairShop_Code/PhoneRepairShop_code.csproj]: No.

File: Customization/T210/PhoneRepairShop_Code/PhoneRepairShop_Code/RSSVRepairPriceMaint.cs (hunk 2 above)
Comment (PetrKorchagin) [reply Customization/T210/PhoneRepairShop_Code/PhoneRepairShop_Code/RSSVRepairPriceMaint.cs]: No.

**Review by k-s-popova — APPROVED** [review]

**Pull request merged**